Automattic/jetpack
# Vdimitrakis/add price in posts endpoint

We need to expose the WooCommerce price in the blaze dashboard list (in Advertising - or in the new  Marketing page)

This change will check if the  `wc_get_product ` method is available and will call for each of the items/posts, it will retrieve the price of each post (if a product) and will inject it in the final results.

This will only work for self-hosted or pressable sites.





Fixes #

## Proposed changes:

*

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






You need to have Woo installed in your testing site and to have (at least) a product created. 
Navigate to the advertising page and check your network inspect tab. Filter requests by /blaze/posts
In the request result, you should see the price data when the post is a product.

![Screenshot 2024-01-17 at 17 01 50](https://github.com/Automattic/jetpack/assets/1416426/a9e466d4-3f66-4ff7-af08-9d4ad24ecd6a)

Then click on the Promote button (on the top right corner of your page). Click continue to load all posts again and check the network for the same call.

you should see the price entry in the posts/results array

posts that are not of product type should have the property but with an empty value (empty string) 


* Go to '..'
*
